libbeat/_meta/config.reference.yml

```diff
@@ -628,6 +628,9 @@ output.elasticsearch:
 # dashboards and index pattern. Example: testbeat-*
 #setup.dashboards.index:
 
+# Force loading of dashboards using Kibana instead of Elasticsearch
```

Comment:
Suggestion: "Force loading of dashboards using the Kibana API without querying Elasticsearch for the version"